code/game/objects/items/weaponry.dm

```diff
@@ -470,6 +470,17 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	desc = "A bust of the famous Greek physician Hippocrates of Kos, often referred to as the father of western medicine."
 	icon_state = "hippocratic"
 
+/obj/item/statuebust/attack_self(mob/living/carbon/user)
+	. = ..()
+	if(.)
+		return
+	user.changeNext_move(CLICK_CD_MELEE)

```

Comment:
can you not, this thing was never really supposed to live too far inside attack chains.

Do you really need a click cooldown on this?

Reply:
If you do, you should use a do_after instead, so the user takes a second or two and has the little progress bar.